oe_whitelabel.theme

```diff
@@ -184,3 +184,61 @@ function oe_whitelabel_preprocess_media__document__default(&$variables) {
     $variables['translations'][] = $wrapper->toFileValueObject();
   }
 }
+
+/**
+ * Implements hook_element_info_alter().
+ */
+function oe_whitelabel_element_info_alter(array &$info): void {
+  $module_handler = \Drupal::service('module_handler');
+
+  if (isset($info['select']) && $module_handler->moduleExists('slim_select')) {
+    $info['select']['#process'][] = '_oe_whitelabel_process_element_select';
```

Comment:
Instead of a `'#process'` callback to add `['#slim_select']` info, we could add the `'#slim_select'` directly here:

```php
$info['select']['#slim_select'] = [];
```

This has the additional advantage of not overwriting existing '#slim_select' settings that were added by other code, e.g. by a `hook_field_widget_complete_options_select_form_alter()`.

Reply:
This is invalid as per https://github.com/openeuropa/oe_whitelabel/issues/235#issuecomment-1802047091 .